hw2/02-reverse.js

```diff
@@ -1,3 +1,21 @@
 /** Exercise 02 - Reverse **/
 
 // Add your code here
+function reverseNumber() {
+    let inputValue = parseInt(document.getElementById("input").value, 10);
```

Comment:
Code is again very nicely written and formatted. After running it produce expected output with one exception: I am unable to reverse 8-digit numbers with leading 0's.

For example: entering `00001111` 
produces `Error: Please input an 8-digit number`
instead of `11110000`.

However, other than that output difference this code is easily readable and efficient.